src/helpviewer/main.cpp

```diff
@@ -17,23 +17,21 @@
  ***************************************************************************/
 #include <iostream>
 #include <QLocale>
-#include <QSettings>
 #include <QTranslator>
 #include <QLibraryInfo>
+#include <qgssettings.h>
 
 #include "qgshelpviewer.h"
 #include "qgsapplication.h"
 #include "qgslogger.h"
-#include "qgsconfig.h"
 
 int main( int argc, char **argv )
 {
-  QgsApplication a( argc, argv, true );
+  QCoreApplication::setOrganizationName( QgsApplication::QGIS_ORGANIZATION_NAME );
+  QCoreApplication::setOrganizationDomain( QgsApplication::QGIS_ORGANIZATION_DOMAIN );
+  QCoreApplication::setApplicationName( QgsApplication::QGIS_APPLICATION_NAME );
 
-  // Set up the QSettings environment must be done after qapp is created
-  QCoreApplication::setOrganizationName( QStringLiteral( "QGIS" ) );
-  QCoreApplication::setOrganizationDomain( QStringLiteral( "qgis.org" ) );
-  QCoreApplication::setApplicationName( QStringLiteral( "QGIS3" ) );
+  QgsApplication a( argc, argv, true );
```

Comment:
The order here is relevant. The application settings appear before crete the application!